### PR TITLE
[TS] LPS-143582

### PR DESCRIPTION
**Pull request by ChrisKian**
https://issues.liferay.com/browse/LPS-143582

Please let me know if you have any questions, thanks!

CC @hongvo2308

### CODE REVIEW

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
The semantic-versioning-jdk8 error is not related to this PR: [LPS-143954](https://issues.liferay.com/browse/LPS-143954)

File: modules/apps/announcements/announcements-web/src/main/resources/META-INF/resources/announcements/configuration.jsp [hunk 1]
```diff
@@ -100,20 +100,18 @@ announcementsPortletInstanceConfiguration = ParameterMapUtil.setParameterMap(Ann
...
-									if (announcementsDisplayContext.isScopeGroupSelected(curGroup)) {
-										leftList.add(new KeyValuePair(String.valueOf(curGroup.getGroupId()), curGroup.getDescriptiveName(locale)));
-									}
-								}
-
 								List<KeyValuePair> rightList = new ArrayList<KeyValuePair>();
 
 								for (Group curGroup : groups) {
-									KeyValuePair tempKeyValuePair = new KeyValuePair(String.valueOf(curGroup.getGroupId()), curGroup.getDescriptiveName(locale));
+									String descriptiveName = curGroup.isOrganization() ? String.format("%s(%s)", curGroup.getDescriptiveName(locale), LanguageUtil.get(request, OrganizationConstants.TYPE_ORGANIZATION)) : curGroup.getDescriptiveName(locale);
```
Comment (adolfopa) [inline modules/apps/announcements/announcements-web/src/main/resources/META-INF/resources/announcements/configuration.jsp:106]: A minor aesthetic issue: shouldn't the format string be `"%s (%s)"` (with a blank before the paren)?

Other than that, i looks good. If that's on purpose (e.g. getDescriptiveName contains trailling blanks), we can forward this to Brian.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/announcements/announcements-web/src/main/resources/META-INF/resources/announcements/configuration.jsp (hunk 1 above)
Comment (ChrisKian) [reply modules/apps/announcements/announcements-web/src/main/resources/META-INF/resources/announcements/configuration.jsp]: Hi @adolfopa, there was no reason for the lack of space.  I've updated the pull to include it, and force-pushed the changes now.

Please let me know if you have any additional questions, thanks!

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
PortalSmoke#Smoke failure should be a flaky issue.

**Comment by Hanlf** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**